test/verify/check-storage-smart

```diff
@@ -91,42 +91,58 @@ class TestStorageSmart(storagelib.StorageSmartCase):
             return
 
         m.upload(["verify/files/smart-dumps"], "/tmp")
+        # Failing disk, storage page is not loaded
+        self.set_smart_dump("Maxtor_96147H8--BAC51KJ0--2", "/dev/sda")
+        self.login_and_go("/system")
+        b.wait_in_text("#smart-status", "1 disk is failing")
+
+        self.set_smart_dump("MCCOE64GEMPP--2.9.09", "/dev/sda")
+        b.wait_not_present("#smart-status")
```

Comment:
Shouldn't we first navigate to the storage overview?  To see the failing disk? Otherwise the test fails a bit weird. Maybe reverse it, first failing, then non-failing, then failing if you don't want to b.go() too often

Reply:
It is... not sure how I missed that.